src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -153,6 +151,45 @@ public ResourceTypeSchema(Schema schema) {
             });
             return null;
         });
+
+        this.unprocessedProperties.computeIfPresent("tagging", (k, v) -> {
+            hasConfiguredTagging = true;
+            ((Map<?, ?>) v).forEach((key, value) -> {
+                if (key.equals("taggable")) {
```

Comment:
nit: Could we have the string as constant.  Those constant values could be defined in the class `ResourceTagging`, which should has all the knowledge of the resource tagging.

Reply:
yeah added those fields